lib/hijack/wrap_observers.js

```diff
@@ -157,18 +162,21 @@ export function wrapMultiplexer (proto) {
 export function wrapForCountingObservers () {
   // to count observers
   let mongoConnectionProto = MeteorX.MongoConnection.prototype;
+
   let originalObserveChanges = mongoConnectionProto._observeChanges;
-  mongoConnectionProto._observeChanges = function (cursorDescription, ordered, callbacks) {
-    let ret = originalObserveChanges.call(this, cursorDescription, ordered, callbacks);
-    // get the Kadira Info via the Meteor.EnvironmentalVariable
-    let kadiraInfo = Kadira._getInfo(null, true);
+
+  mongoConnectionProto._observeChanges = async function (cursorDescription, ordered, callbacks) {
+    let ret = await originalObserveChanges.call(this, cursorDescription, ordered, callbacks);
+
+    let kadiraInfo = Kadira._getInfo();
```

Comment:
If I remember correctly there was no actual difference, and I ended up simplifying. The `EnvironmentVariable` mechanism seemed to abstract away what we already did directly to the current fiber. It's like everything now is analogous to what `EnvironmentVariable` was before. At one point the new `EnvironmentVariable` mechanism in Meteor 3 was unstable too and I ended going back to ALS, as it would be just one more additional layer prone to problems.

In short, if there was a reason it was not documented anywhere, seems like the interrupted migration to a newer syntax or something to me.